crates/core/tedge/src/cli/connect/bridge_config_c8y.rs

```diff
@@ -57,7 +57,6 @@ impl From<BridgeConfigC8yParams> for BridgeConfig {
             r#"measurement/measurements/create out 2 c8y/ """#.into(),
             r#"event/events/create out 2 c8y/ """#.into(),
             r#"alarm/alarms/create out 2 c8y/ """#.into(),
-            r#"devicecontrol/notifications in 2 c8y/ """#.into(),
```

Comment:
This change can safely be included in the release without being reverted.

Reply:
I'll create a follow-up PR to include some stuff. Also, the bug fix (duplicated "c8y" on cmd ID) is going to be reverted.